deploy/olm-catalog/knative-eventing-operator/0.6.0/knative-eventing-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,199 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: knative-eventing-operator
+      deployments:
+      - name: knative-eventing-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: knative-eventing-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: knative-eventing-operator
+            spec:
+              containers:
+              - command:
+                - knative-eventing-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: knative-eventing-operator
+                image: quay.io/openshift-knative/knative-eventing-operator:v0.5.0
```

Comment:
This should be 0.6.0, and should've carried over from `deploy/operator.yaml`, so I'm assuming you generated this _before_ updating `operator.yaml`?

Reply:
@jcrossley3  yes - that's exactly right :-)   and now I fixed it